examples/renderable/main.rs

```diff
@@ -347,18 +374,28 @@ fn run() -> Result<(), Error> {
     );
 
     let display_config = DisplayConfig::load(display_config_path);
-    let pipeline_builder = Pipeline::build().with_stage(
-        Stage::with_backbuffer()
-            .clear_target([0.0, 0.0, 0.0, 1.0], 1.0)
-            .with_pass(DrawShaded::<PosNormTex>::new()),
-    );
 
-    let mut game = Application::build(resources_directory, Example)?
+
+    let game = Application::build(resources_directory, Example)?
         .with::<ExampleSystem>(ExampleSystem, "example_system", &[])
+        .with_frame_limit(FrameRateLimitStrategy::Unlimited, 0)
```

Comment:
Remove this maybe?

Reply:
I was using this to benchmark the example's performance so I could see how text rendering impacted the FPS.  I decided to leave it because it's probably nice to show off the frame rate anyways on an example called `renderable`.  What do you think?

Reply:
As long as peoples computers don't burn to cinders I'm fine with it :P

Reply:
Well, mine didn't :D